src/Components/ProductCard/ProductCard.jsx

```diff
@@ -17,18 +18,20 @@ export const ProductCard = () => {
                 {wishList.some((product) => product._id === item._id) ? (
                   <BsFillHeartFill
                     className="wishList-icon"
-                    onClick={() =>
+                    onClick={() => {
                       dispatch({
                         type: "REMOVE_FROM_WISHLIST",
                         payload: item,
-                      })
-                    }
+                      }),
+                        toast("removed from wishlist", { icon: "❌" });
+                    }}
```

Comment:
If you are calling two or more function in` onClick` then use a `clickHandler` function.